frontend/components/community/activity.tsx

```diff
@@ -92,25 +99,31 @@ export default function ActivityDisplay({ data }: { data: Activity[] }) {
       <React.Fragment>
          <Heading
             as="h3"
-            fontSize="var(--font-size-5)"
+            fontSize={`${fontSize['3xl']}`}
+            color={`${color.gray[900]}`}
             margin={{
-               base: 'var(--space-7) 0 var(--space-5)',
-               md: 'var(--space-10) 0 var(--space-7)',
+               base: `${space[7]} 0 ${space[5]}`,
+               md: `${space[10]} 0 ${space[7]}`,
             }}
          >
             Latest Community Activity
          </Heading>
          {data ? (
-            <SimpleGrid columns={{ base: 1, sm: 2, md: 3 }} spacing={7}>
-               {data.map((activity, i) => (
-                  <ActivityCard key={i} data={activity} />
+            <SimpleGrid columns={{ base: 1, sm: 2, md: 3 }} spacing={5}>
```

Comment:
Was it intentional to change the 7 to a 5?  Everything else seemed to map 1:1 so just wanted to check :)

Reply:
Double checked, and yup! There were a few places that I had to adjust the styling slightly to get it to match up. 

Should have made that more clear in commits, my bad! Thanks for checking